pkg/users/group-details.js

```diff
@@ -0,0 +1,185 @@
+
+        const current_account = accounts.find(a => a.name == account);
+        if (current_account.loggedIn)
+            setLoggedInUsersAffected(true);
+
+        return cockpit.spawn(["gpasswd", "-d", account, group], { superuser: "require", err: "message" })
+                .then(() => {
+                    setSelected(selected.filter(item => item !== account));
+                    setIsOpenAccount(false);
+                }, show_unexpected_error);
+    };
+
+    const addAccount = (account, isUndo) => {
+        if (!isUndo)
+            setHistory([...history, { type: 'added', name: account }]);
+
+        const current_account = accounts.find(a => a.name == account);
+        if (current_account.loggedIn) setLoggedInUsersAffected(true);
+
+        return cockpit.spawn(["gpasswd", "-a", account, group], { superuser: "require", err: "message" })
+                .then(() => {
+                    setSelected([...selected, account]);
+                    setIsOpenAccount(false);
+                }, show_unexpected_error);
+    };
+
+    const onSelectAccount = (event, selection) => {
+        if (selected.includes(selection)) {
+            removeAccount(selection);
+        } else {
+            addAccount(selection);
+        }
+    };
+
+    const chipGroupComponent = () => {
+        return (
+            <LabelGroup numLabels={10}>
+                {(selected || []).map((currentLabel, index) => {
+                    const optional = !current_group.userlistPrimary.includes(currentLabel) && superuser.allowed ? { onClose: () => removeAccount(currentLabel) } : {};
+
+                    return (
+                        <Label key={currentLabel}
+                               color={accounts.find(ac => ac.name === currentLabel).isAdmin ? "gold" : "cyan"}
+                               {...optional}
+                        >
+                            {currentLabel}
+                        </Label>
+                    );
+                })}
+            </LabelGroup>
+        );
+    };
+
+    return (
+        <FormGroup
+            fieldId="group-accounts"
+            helperText={
+                (history.length > 0)
+                    ? <HelperText className="pf-c-form__helper-text">
+                        <Flex>
+                            {loggedInUsersAffected && <HelperTextItem id="group-accounts-helper" variant="warning">{_("The logged in users must log out and log back in for the new configuration to take effect.")}</HelperTextItem>}
+                            {history.length > 0 && <Button variant="link" id="group-undo-btn" isInline icon={<UndoIcon />} onClick={undoAccountChanges}>{_("Undo")}</Button>}
+                        </Flex>
+                    </HelperText>
+                    : ''
+            }
+            id="group-accounts-form-group"
+            label={_("Accounts")}
+            validated={history.length > 0 ? "warning" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230215-180016-4889e9dc-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)